docs/tuning-apps/coll-tuned.rst

```diff
@@ -0,0 +1,387 @@
+Tuning Collectives
+==================
+
+Open MPI's `coll` framework provides a number of components implementing
```

Comment:
In the rest of the docs, I think the convention we've used is
```
``coll``
```

i.e., double quotes so that `coll` shows up in fixed-width font, not italics.

Same with the coll component names, below.  Like this:

![Screenshot 2024-06-28 at 3 36 21 PM](https://github.com/open-mpi/ompi/assets/617316/fa962886-f142-4b25-a7ca-d9df5ac353bc)

Reply:
OK, fixed that